apps/main-landing/src/components/products-section/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,48 @@
+import { classes } from '@idriss-xyz/ui/utils';
+import Link from 'next/link';
+
+import { TabOption } from '../../types';
+
+type Properties = {
+  options: TabOption[];
+  activeOptionKey: string;
+  asLink: boolean;
+};
+
+export const Tabs = ({ activeOptionKey, options, asLink }: Properties) => {
+  return (
+    <div className="flex items-start gap-1 rounded-[50px] bg-[#022218] p-1 text-label5 lg:text-label4">
+      {options.map((option) => {
+        return asLink ? (
+          <Link
+            href={`/#${option.key}`}
+            passHref
+            legacyBehavior
+            key={option.key}
+          >
+            <a
+              className={classes(
+                'flex cursor-pointer items-start rounded-[100px] bg-[#17ff4a1a] px-[9.5px] py-2 text-midnightGreen-100 transition-colors duration-1000 lg:px-4',
+                activeOptionKey === option.key &&
+                  'bg-mint-400 text-neutralGreen-900',
+              )}
+            >
+              {option.name}
+            </a>
+          </Link>
+        ) : (
+          <span
+            key={option.key}
+            className={classes(
+              'flex items-start rounded-[100px] bg-[#17ff4a1a] px-[9.5px] py-2 text-midnightGreen-100 lg:px-4',
+              activeOptionKey === option.key &&
+                'bg-mint-400 text-neutralGreen-900',
+            )}
+          >
```

Comment:
this1 is missing transitions from the link version, also maybe you can extract similar classnames to a variable? Feels like link just needs `cursor-pointer` but maybe it's not needed since links should have cursor pointer out of the box

Reply:
iirc, we agreed that these pills will be read-only on mobile to avoid misclick navigation, that's why we have this separation